README.md

```diff
@@ -24,6 +68,50 @@ The `pages/api` directory is mapped to `/api/*`. Files in this directory are tre
 
 This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
 
```

Comment:
deployment on vercel wala line hata ke bas ye likh do ki you can check it out the link